test/runtests.jl

```diff
@@ -48,64 +40,77 @@ for dim in (1,2,3)
             @test (@inferred t_sym + t_sym) == 2*t_sym
             @test (@inferred -t_sym) == zero(t_sym) - t_sym
             @test (@inferred 2*t_sym) == t_sym*2
-
             @test 2*t_sym == t_sym*2
-
             @test (@inferred rand(t_sym) * 0.0) == zero(t_sym)
         end
     end
 end
 end # of testset
 
-
 ############
 # Indexing #
 ############
 @testset "Indexing" begin
 for dim in (1,2,3)
     for order in (1,2,4)
-        n_sym = n_independent_components(dim, true)
-        n = n_independent_components(dim, false)
-
         if order == 1
-            vec = Tensor{order, dim}(rand(dim))
-            if dim == 1
-                @test_throws BoundsError vec[2]
-            elseif dim == 2
-                @test_throws BoundsError vec[3]
-            elseif dim == 3
-                @test_throws BoundsError vec[4]
-            else
-                for t in (SymmetricTensor(rand(n), Val{dim}), Tensor(rand(n), Val{dim}))
```

Comment:
This was some really old constructors of tensors, eh? :)

Due to the `if order == 1` above, only order 1 was tested btw